internal/backend/sessions/sessioncalls/calls.go

```diff
@@ -47,12 +56,28 @@ func New(z *zap.Logger, config Config, svcs *sessionsvcs.Svcs) Calls {
 	opts := config.Temporal.Worker
 	opts.DisableRegistrationAliasing = true
 	opts.OnFatalError = func(err error) { z.Error("temporal worker error", zap.Error(err)) }
+	opts.DisableWorkflowWorker = true
+
+	uniqueWorkerCallTaskQueueName := fmt.Sprintf("%s_%s", taskQueueName, strings.Replace(uuid.NewString(), "-", "", -1))
```

Comment:
If this is unique per `New`, what happens when the server is restarted and  there are tasks in this queue?

Reply:
now commented.